packages/riipen-ui-docs/src/pages/components/popover/Demo.jsx

```diff
@@ -0,0 +1,217 @@
+                color="primary"
+                checked={state.anchorPosition.vertical === "center"}
+              />
+              <Radio
+                label="Bottom"
+                value="bottom"
+                color="primary"
+                checked={state.anchorPosition.vertical === "bottom"}
+              />
+            </RadioGroup>
+            <RadioGroup
+              label="Anchor Horizontal"
+              hint=""
+              onChange={handleAnchorHorizontal}
+              required
+            >
+              <Radio
+                label="Left"
+                value="left"
+                color="primary"
+                checked={state.anchorPosition.horizontal === "left"}
+              />
+              <Radio
+                label="Center"
+                value="center"
+                color="primary"
+                checked={state.anchorPosition.horizontal === "center"}
+              />
+              <Radio
+                label="Right"
+                value="right"
+                color="primary"
+                checked={state.anchorPosition.horizontal === "right"}
+              />
+            </RadioGroup>
+            <RadioGroup
+              label="Content Vertical"
+              hint=""
+              onChange={handleContentVertical}
+              required
+            >
+              <Radio
+                label="Top"
+                value="top"
+                color="primary"
+                checked={state.contentPosition.vertical === "top"}
+              />
+              <Radio
+                label="Center"
+                value="center"
+                color="primary"
+                checked={state.contentPosition.vertical === "center"}
+              />
+              <Radio
+                label="Bottom"
+                value="bottom"
+                color="primary"
+                checked={state.contentPosition.vertical === "bottom"}
+              />
+            </RadioGroup>
+            <RadioGroup
+              label="Content Horizontal"
+              hint=""
+              onChange={handleContentHorizontal}
+              required
+            >
+              <Radio
+                label="Left"
+                value="left"
+                color="primary"
+                checked={state.contentPosition.horizontal === "left"}
+              />
+              <Radio
+                label="Center"
+                value="center"
+                color="primary"
+                checked={state.contentPosition.horizontal === "center"}
+              />
+              <Radio
+                label="Right"
+                value="right"
+                color="primary"
+                checked={state.contentPosition.horizontal === "right"}
+              />
+            </RadioGroup>
+            <Button onClick={buttonClick}>Open Popover</Button>
+          </Form>
+        </div>
+        <div className="half" ref={ref => (state.anchorEl = ref)}>
```

Comment:
instead of saving the ref in state you should use the "useRef" hook https://reactjs.org/docs/hooks-reference.html#useref!

Reply:
ill take a look, thanks :)